middleware/grpcmiddleware.go

```diff
@@ -0,0 +1,29 @@
+package middleware
+
+import (
+	"errors"
+	"runtime"
+
+	"github.com/tron-us/go-common/v2/log"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	grpc_recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
+)
+
+var (
+	// Panic handler prints the stack trace when recovering from a panic.
+	RecoveryCustomFunc grpc_recovery.RecoveryHandlerFunc = grpc_recovery.RecoveryHandlerFunc(func(p interface{}) error {
+		buf := make([]byte, 1<<16)
```

Comment:
for your consideration: https://golang.org/pkg/bytes/#Buffer
if it can help to make the code cleaner

Reply:
ok.